modules/openy_gc_storage/openy_gc_storage.post_update.php

```diff
@@ -90,3 +90,165 @@ function openy_gc_storage_post_update_migrate_eventseries_instructors(&$sandbox)
 function openy_gc_storage_post_update_migrate_eventinstance_instructors(&$sandbox) {
   _openy_gc_storage_migrate_instructors($sandbox, 'eventinstance', 'field_ls_host_name');
 }
+
+/**
+ * Create terms for Virtual Y Duration.
+ */
+function openy_gc_storage_post_update_create_durations(&$sandbox) {
```

Comment:
The function is called once, so probably we just can do this all in the post-update itself?